reward/src/lib.rs

```diff
@@ -0,0 +1,534 @@
+use std::ffi::OsString;
+
+use anyhow::anyhow;
+
+use ethers::prelude::{Address, SignerMiddleware, U256};
+
+use librad::git::identities;
+use librad::git::identities::SomeIdentity::Person;
+use librad::PeerId;
+
+use std::path::Path;
+use std::str::FromStr;
+
+use std::collections::HashMap;
+use std::sync::Arc;
+
+use rad_ens::resolver::PublicResolver;
+use radicle_common::args::{Args, Error, Help};
+use radicle_common::ethereum::{ProviderOptions, SignerOptions};
+use radicle_common::{ethereum, profile};
+use radicle_common::{git, keys};
+use radicle_terminal as term;
+
+use crate::erc_20::ERC20;
+use crate::superseeder::Superseeder;
+
+mod erc_20;
+mod superseeder;
+
+pub const HELP: Help = Help {
+    name: "reward",
+    description: env!("CARGO_PKG_DESCRIPTION"),
+    version: env!("CARGO_PKG_VERSION"),
+    usage: r#"
+Usage
+
+    rad reward [<range>] [<option>...]
+
+    range defaults to 'rad/reward..' but you can use any range you want.
+
+Options
+    --dry                        Shows summary and skips signing TXs
+    --currency                   wETH, DAI, USDC, USDT, or any other ERC-20 Address
+    --amount                     Should be an integer of currency's smallest denomination
```

Comment:
This is ok for now, though UX-wise it's not going to be nice to have very large numbers here

Reply:
you're talking about `--amount`, right? this is for test and automation purposes, what other option do we have?